tests/test_api.py

```diff
@@ -106,6 +123,41 @@ def csv_col_mismatch():
     assert jsonres['ok']
 
 
+@pytest.mark.asyncio
+async def test_apify_hour_format(rmock, csv_hour, client):
+    content = csv_hour.replace('<sep>', ';').encode('utf-8')
+    url = 'http://example.com/file.csv'
```

Comment:
Ok to generate URLs in tests?

Reply:
👍 we could even use `faker` to ensure they are unique

Reply:
It was not downloaded as a dependency yet. Did not introduce it just for this.